src/main/java/ladder/LadderGame.java

```diff
@@ -0,0 +1,12 @@
+package ladder;
+
+import ladder.controller.LadderController;
+
+public class LadderGame {
+    public static void main(String[] args) {
+        LadderController ladderController = new LadderController();
```

Comment:
LadderController는 인스턴스화가 필요하지 않은 형태로 구현하셨으니 
run 메서드를 바로 호출하면 어떨까요?
```suggestion
        LadderController.run();
```